pinot-common/src/main/java/com/linkedin/pinot/startree/hll/HllConfig.java

```diff
@@ -118,4 +122,10 @@ public int getHllFieldSize() {
     }
     return derivedHllFieldToOriginMap;
   }
+
```

Comment:
Please check with @antumbde if this can be moved here or if there's a better solution.